ni_measurementlink_service/_internal/discovery_client.py

```diff
@@ -204,12 +206,63 @@ def resolve_service(self, provided_interface: str, service_class: str = "") -> S
 
 def _get_discovery_service_address() -> str:
     key_file_path = _get_key_file_path()
+    _ensure_discovery_service_started(key_file_path)
     _logger.debug("Discovery service key file path: %s", key_file_path)
     with _open_key_file(str(key_file_path)) as key_file:
         key_json = json.load(key_file)
         return "localhost:" + key_json["InsecurePort"]
 
 
+def _ensure_discovery_service_started(key_file_path: pathlib.Path):
```

Comment:
Type hint
```suggestion
def _ensure_discovery_service_started(key_file_path: pathlib.Path) -> None:
```

(otherwise it is assumed to return `typing.Any`)